standard.md

```diff
@@ -0,0 +1,94 @@
+# The definition of the standard
+
+## Glossary
+
+There are three levels of definitions in this standard:
+
+- **Must**: all diagnostic have to meet this conditions to comply with the standard.
+- **Recommend**: diagnostics are encouraged to meet them but they are not mandatory. Nevertheless, if a list of compatible diagnostics is compiled, those who will meet them will be highlighted.
+- **Suggest**: they will never ve enforced, but the team consider them as nice to have. They mostly correspond to quality of life improvements.
+
+In the same way, when defining the content of the yaml files, three different levels of definitions are  used:
+
+- **Required**: they must allways be present with the defined meaning and allowed values.
+- **Reserved**: they can be omitted, but in the case that are present, they must have the defined meaning and allowed values.
```

Comment:
I don't think all of them should be able to go without the reserved keys they needed. Remember that we are just defining them to avoid confusion and that, even for most diagnostic they will be informative, for others they may be key information (I am thinking of a diagnostic to compare members within a single simulation, for example)

Reply:
Another thing I think i will add to this is that diagnostic should be resilient to the presence of reserved keys that they do not use. That ways tools can add all the reserved keys they want without the fear of diags failing because of that. That will be handy for metadata

Reply:
Fine for me